Tiltfile

```diff
@@ -49,6 +49,16 @@ if live_reload:
 else:
   docker_build(project_name, '.')
 
-k8s_yaml(helm('./charts/newrelic-infrastructure', name='nr', values=['values-dev.yaml', 'values-local.yaml']))
+# ns_yaml_str is wrapped as Blob so that Tiltfile will treat it as DATA and not as a filepath
+ns_yaml_str = """
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: nri-k8s-dev
+"""
+k8s_yaml([blob(ns_yaml_str)])
```

Comment:
E2E testing also uses the `e2e-resources`, so I moved the namespace creation to the Tiltfile to keep these changes contained to the local dev env.